interpreter/cling/lib/Interpreter/DynamicLibraryManagerSymbol.cpp

```diff
@@ -541,6 +542,25 @@ namespace cling {
       }
     }
 
+    // On Windows, the symbols are present in COFF format.
```

Comment:
Can you move the comment in the condition and stick the `else if` closer?

In addition we should have the same logic for ignoring symbols as we have for ELF -- we need essentially the same code only the loop is over export_directory_begin/end.

Reply:
I think the inner code is similar? (the logic of ignoring symbols)

Reply:
I fail to see something like:

```
  uint32_t Flags = S.getFlags();
  // DO NOT insert to table if symbol was undefined
  if (Flags & llvm::object::SymbolRef::SF_Undefined)
      continue;
```

in the newly added code...

Reply:
Addressed in a comment...